components/build-templates/staging/e2e-registryredhatio-pull-secret.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
we are running e2e-tests in production cluster only, does it make sense to duplicate in staging as well? Are you keeping it for any future use case?

Reply:
Even if we keep it, we need to have a comment (here and in Vault if possible) that the credentials are not used now.

Reply:
yeah, I would remove it as part of the refactoring story.

Reply:
@tisutisu you had commented this thought to the rotation document and we talked there already. The story mentions staging. It's up to you. If it is really unnecessary, I'll remove the update to staging.